main.go

```diff
@@ -243,10 +243,10 @@ func getFromMetadata(urlStr string) ([]byte, error) {
 }
 
 type config struct {
-	XdsServers               []server                             `json:"xds_servers,omitempty"`
-	Node                     *node                                `json:"node,omitempty"`
-	CertificateProviders     map[string]certificateProviderConfig `json:"certificate_providers,omitempty"`
-	GRPCServerResourceNameID string                               `json:"grpc_server_resource_name_id,omitempty"`
+	XdsServers                         []server                             `json:"xds_servers,omitempty"`
+	Node                               *node                                `json:"node,omitempty"`
+	CertificateProviders               map[string]certificateProviderConfig `json:"certificate_providers,omitempty"`
```

Comment:
The above 3 lines don't contain an actual change - just tab/whitespace changes?

Reply:
That's `gofmt`s work.